script.js

```diff
@@ -29,4 +29,211 @@ aboutLink.addEventListener('click', () => {
 contactLink.addEventListener('click', () => {
   mobileMenu.classList.replace('show_menu', 'hide_menu');
   header.classList.replace('hide_menu', 'show_menu');
-});
\ No newline at end of file
+});
+
+// Array of project objects to load the content dynamically
+const works = [
+  {
+    id: 'featureWork',
+    name: 'Multi-Post Stories',
+    description: `A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard dummy text
+                  ever since the 1500s, when an unknown printer took a standard
+                  dummy text.`,
+    featImage: 'imgs/img-placeholder-fw.png',
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveLink: 'https://luigirazum.github.io/',
+    sourceLink: 'https://github.com/luigirazum/mv-portfolio-project',
+  },
+  {
+    id: 'work1',
```

Comment:
- [ ] Great job so far. please for most of your projects, ensure the see live and see source buttons are well placed and visible
- [ ] the see live and see source buttons are not visible in most desktop popups. please fix that.
your desktop popup
![Screenshot (597)](https://user-images.githubusercontent.com/65298048/202639466-a936fa27-86dd-4128-8d88-7d928c1ccaf6.png)
figma design
![Screenshot (599)](https://user-images.githubusercontent.com/65298048/202639578-ee610967-1574-425b-9474-e655dc1a189b.png)


your mobile version
![Screenshot (596)_LI](https://user-images.githubusercontent.com/65298048/202639817-6f07c0fb-8899-4730-97f1-9c935d06f01f.jpg)
figma version
![Screenshot (600)](https://user-images.githubusercontent.com/65298048/202639962-84cb863b-2d70-4c66-86d0-194c4b1a760c.png)